css/style.css

```diff
@@ -6,3 +6,124 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+/*General style*/
+body {
+    background-color: lightgray;
+}
+
+/*Navigation style*/
+.nav_bar {
+    width: 100%;
+    height: 50px;
+    background-color: #686868;
+}
+
+.intro {
+    margin: 20px;
+    margin-left: 100px;
+    line-height: 50px;
+    font-family: Lucida Handwriting;
+    font-size: 30px;
+    color: darkgreen;
+}
+
+.menu {
+    float: right;
+}
+
+.list {
+    list-style: none;
+    display: inline-block;
+    margin: 0 30px;
+    line-height: 20px;
+    font-size: 20px;
+}
+
+.link {
+    color: black;
+    list-style: none;
+    text-decoration: none;
+}
+
+.link:hover {
+    color: orangered;
+}
+
+/*Hero Style*/
+.home{
+    display: flex;
+    flex-direction: row;   
+}
+.image{
+    margin-left: 200px;
+    margin-top: 50px;
+    border-radius: 6000px;
+    border-style: solid;
+    border-color:#d0d0d0;
+}
+.text{
+    margin-left: 100px;
+    margin-top: 100px;
+    text-align: center;
+    justify-content: flex-end
+}
+h1{
+    text-align: center;
+}
+.p1{
+    font-size: 16px;
+}
+.article{
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    margin: 20px 100px;
+    font-size: 20px;
+    
+}
+.article1{
+    border-style: solid;
+    border-color: #a0a0a0;
+    border-radius: 16px;
+    margin-top: 50px;
+    padding: 30px;
+
+}
+.article2 {
+    border-style: solid;
+    border-color: #a0a0a0;
+    border-radius: 16px;
+    margin-top: 50px;
+    padding: 30px;
+}
+.article3 {
+    border-style: solid;
+    border-color: #a0a0a0;
+    border-radius: 16px;
+    margin-top: 50px;
+    padding: 30px;
+}
```

Comment:
It looks like all three articles have the same styling. If this is deliberate, it would have been better to give them all the same class name and applied one set of styles. The reason for this is that if you wanted to change the border-style, for example, you would currently need to change it in three separate places to keep the styles the same. Now imagine if there were 20 articles and you had to make changes 20 times.